apps/widhwbttm/widhwbttm.wid.js

```diff
@@ -35,11 +36,12 @@
             v_switch=0;
           }
        }    
-  g.reset().setFontVector(v_font_size).setFontAlign(-1, 0).setColor("#0ff");
-  //clean a longer previous string
-  g.drawString("                        ", this.x, this.y+11, true);
+  g.reset().setFontVector(v_font_size).setFontAlign(-1, 0); 
+  //clean a longer previous string, care with br widgets
+  g.drawString("               ", this.x, this.y+11, true);
   g.drawString(v_str_hw, this.x, this.y+11, true);  
   } //end draw
 
 WIDGETS["wdhwbttm"]={area:"bl",width:Bangle.CLOCK?0:60,draw:draw};
```

Comment:
This looks wrong here actually - why is the width dependent on Bangle.CLOCK? And isn't the whole thing a lot wider than 60px anyway?

Reply:
Iong strings were the cause, but I'll trim and convert into shorter measure units. Mistakes from an amateur with hurry :|

Reply:
BTW, i just copied this widget parameters (width:Bangle.CLOCK?0:60,) from the v.004  of widclkbttm. I was assuming that it hides the widget when a clock is open

Reply:
Ahh, yes - but you also need to set the width to the actual width of the widget, or you'll end up overlapping with other widgets that want to draw in the same area ('bl'). It looks a lot more than 60 here - maybe 160 at least on Bangle.js 1?

... and if you're changing the font between Bangles you probably need to change that width too